src/App.js

```diff
@@ -1,13 +1,42 @@
-import Calculator from './components/Calculator';
 import './App.css';
```

Comment:
- [ ] When running the app there is an error(see below screenshot). Because there is no App.css file in your project directory. Kindly fix it and request another review. 
![Screenshot (258)](https://user-images.githubusercontent.com/67087664/165363509-66d16e81-a3e8-4186-b292-1a99d42b2e0b.png)